Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -217,10 +219,14 @@ private static async Task ProcessYamlFile(string filePath, string targetPath, Di
                             {
                                 line = $"  m_Script: {{fileID: {tuple.Item2}, guid: {tuple.Item1}, type: 3}}";
                             }
+                            else if (nonClassDictionary.Contains(filePath))
```

Comment:
This is supposed to be the guid, not the yaml file path, right?

Reply:
yeah, that makes sense